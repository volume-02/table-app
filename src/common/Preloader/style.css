.preloader {
  border: 6px solid #f3f3f3;
  border-top: 6px solid rgba(43, 128, 167, 0.911);
  border-radius: 50%;
  width: 90px;
  height: 90px;
  animation: spin 2s linear infinite;
}

.preloaderContainer {
  margin-top: 10rem;
  display: flex;
  width: 100%;
  justify-content: center;
}

@keyframes spin {
  0% {
    transform: rotate(0deg);
  }
  100% {
    transform: rotate(360deg);
  }
}
