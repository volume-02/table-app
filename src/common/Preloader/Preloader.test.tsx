import React from 'react';
import Preloader from './Preloader';
import { shallow } from 'enzyme';

describe('Preloader', () => {
  it('Should render preloader', () => {
    const component = shallow(<Preloader />);
    expect(component).toMatchSnapshot();
  });

  it('hello', () => {
    const foo = shallow(
      <h1>
        <div>hello</div>
      </h1>
    );
    expect(foo).toMatchSnapshot();
  });
});
