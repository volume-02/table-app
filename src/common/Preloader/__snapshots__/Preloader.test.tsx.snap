// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preloader Should render preloader 1`] = `
<div>
  <div />
</div>
`;

exports[`Preloader hello 1`] = `
<h1>
  <div>
    hello
  </div>
</h1>
`;
